cime/src/drivers/mct/main/seq_domain_mct.F90

```diff
@@ -672,11 +672,13 @@ subroutine seq_domain_check_grid(dom1, dom2, attr, eps, mpicom, mask)
           diff = abs(data1(n)-data2(n))
           max_diff = max(max_diff,diff)
           if (diff > eps) then
-      !debug            write(logunit,*)'n= ',n,' data1= ',data1(n),' data2= ',data2(n),' diff= ',diff, ' eps= ',eps
+             ! debug: write(logunit,*)'n= ',n,' data1= ',data1(n),' data2= ',data2(n),' diff= ',diff, ' eps= ',eps
+             write(logunit,150) n,data1(n),data2(n),diff,eps
```

Comment:
Did you want to keep this or have it be included only with a DEBUG type flag?  I'm happy to keep it, just curious since it was originally commented out, so I assume you made this change to check your progress.

Reply:
I think we should keep it like this. If this gets triggered then the model is going to crash either way, so it's good to have this information in the log file. I should probably just delete the old version that I commented out.